Attachments/adding-attachments.md

```diff
@@ -0,0 +1,251 @@
+## How to upload and attach files from a third party applications
```

Comment:
Change this to either `...from a third party application` or `...from third party applications`